ecl/hql/hqlfold.cpp

```diff
@@ -2154,6 +2154,41 @@ IHqlExpression * foldConstantOperator(IHqlExpression * expr, unsigned foldOption
             }
             break;
         }
+    case no_regex_findset:
+        {
+            IValue * t0 = expr->queryChild(0)->queryValue();
+            IValue * t1 = expr->queryChild(1)->queryValue();
```

Comment:
picky:  Not something to change, but variables that begin with t suggest a type.  I would have e for an expression v for a value (as in this case) and t for a type.  Not worth changing.  (I didn't the code this was derived from!)